docs/topics/articles.md

```diff
@@ -0,0 +1,184 @@
+# `Article`s and `ListingPage`s
+
+`Article`s are publishable content types that live in collections. Normally
+articles are mounted under a parent's URL.
+
+An example of an article is a Press Release, which would be mounted under a
+Press Release Listing Page. The Listing Page has the URL "/press" and thus all
+the Press Releases have the URL "/press/<slug>".
+
+## `ArticleBase`
+
+The `icekit.articles.models.ArticleBase` inherits `title`, `slug` and
+publishing fields, and each `ArticleBase` subclass implements some key
+functionality:
+
+1. **Links to parents.**
+In order for an Article to know its URL (which is based on a parent's URL), it
+should define a `parent` link, which is normally a link to a `ListingPage`.
```

Comment:
I meant faked in the sense that some objects implement `parent` as a property that fetches an object to act as parent via some other mechanism (e.g. just picking the first one it finds) instead of having a parent FK.

There are one time setup steps for the end user in both cases. Create and publish a list page. Maybe it has a category field maybe it doesn't, but this difference shouldn't be a significant hurdle for users.

I'm not sure what you mean by the page needing the same slug as the category.

Once there's a published list page, they just create an article and select either an appropriate category or a parent instance. I don't see a huge difference for the user. It's probably easier to select a category from a select list than search for a specific list page in the page tree.

My reluctance to go ahead with this FK to a parent page right now is that I think it will be difficult to change or remove once anything is using it, and I think it's still not clear that this is definitely the right way to go.